index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>What's Popular Where</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <script src="main.js"></script>
+</head>
+<body>
+
+        <!--MAIN HEADER-->
+    <div id="header-container">
+        <header>
+            <h1>Popular Tweets by Location</h1>
+            <ul>
+                <li><a href="about.html">About</a></li>   <!-- links to about page!-->
+            </ul>
+        </header>
+        </div>
+
+        <!--MAP AND GEOLOCATION DISPLAY/DATA-->
+    <div class="map-data">
+
+    </div>
+
+        <!--SEARCH FOR TWEETS AND OTHER DATA DISPLAYED-->
+        <div class="search-container">
```

Comment:
At some point this should probably be a form with inputs.  But for now, as long as there are id's, this should work.